h2/src/main/org/h2/expression/SequenceValue.java

```diff
@@ -5,6 +5,8 @@
  */
 package org.h2.expression;
 
+import java.math.BigDecimal;
+
```

Comment:
What is that? A merge glitch?

Reply:
That's the only explanation. Normally the import optimization would have removed that.

Reply:
Did some resolving of mergeconflicts in this file as well.